Dockerfile

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM quay.io/eclipse/che-container-tools:1.0.0-8caea0f
+
+ENV KUBECTL_VERSION v1.17.0
+ENV HELM_VERSION v3.0.2
+ENV HOME=/home/theia
+ENV BZL_VERSION=3.2.0
+ENV BUIDLERS_VERSION=3.0.0
+ENV JAVA_VERSION=1.8.0
+ENV JAVA_ARCH=x86_64
+
+RUN mkdir /projects && mkdir -p /home/theia && \
+    # Change permissions to let any arbitrary user
+    for f in "${HOME}" "/etc/passwd" "/projects"; do \
+      echo "Changing permissions on ${f}" && chgrp -R 0 ${f} && \
+      chmod -R g+rwX ${f}; \
+    done && \
+    # buildah login requires writing to /run
+    chgrp -R 0 /run && chmod -R g+rwX /run && \
+    curl https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl && \
+    chmod +x /usr/local/bin/kubectl && \
+    curl -o- -L https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz | tar xvz -C /usr/local/bin --strip 1 && \
```

Comment:
You don't need `helm` and `kubectl` in this image, it's part of [che-container-tools](https://github.com/che-dockerfiles/che-container-tools/blob/master/Dockerfile)